src/components/navigation/AppNavigationBoard.vue

```diff
@@ -82,7 +108,7 @@
 </template>
 
 <script>
-import { AppNavigationIconBullet, AppNavigationCounter, AppNavigationItem, ColorPicker, Actions, ActionButton } from '@nextcloud/vue'
+import { AppNavigationIconBullet, AppNavigationCounter, AppNavigationItem, ColorPicker, Actions, ActionButton, ActionRadio } from '@nextcloud/vue'
```

Comment:
Import components directly to gain some bytes?

Reply:
When I checked last time we used so many of them that the combined bundle was actually smaller than including each of them, but i check that in irregular intervals.